runtime/src/bank/tests.rs

```diff
@@ -611,7 +611,7 @@ fn store_accounts_for_rent_test(
     }
 }
 
-fn create_child_bank_for_rent_test(root_bank: &Bank, genesis_config: &GenesisConfig) -> Bank {
+fn create_child_bank_for_rent_test(root_bank: Arc<Bank>, genesis_config: &GenesisConfig) -> Bank {
     let mut bank = Bank::new_from_parent(
         root_bank,
         &Pubkey::default(),
```

Comment:
I am confused.
`Bank::new_from_parent()` used to take `&Arc<Bank>`, but now takes `Arc<Bank>`.
How did the previous version compiled?